rust/opendp_tooling/src/codegen/python.rs

```diff
@@ -131,6 +131,7 @@ We suggest importing under the conventional name ``dp``:
             "{}{}",
             "The ``transformations`` module provides functions that deterministicly transform datasets.",
             special_boilerplate("transformations".to_string())),
+        "internal" => "The ``internal`` module provides functions that can be used to construct library primitives without the use of the \"honest-but-curious\" flag.".to_string(),
```

Comment:
Should it be `_internal`? Or is it even rendered by Sphinx? Whatever works.

Reply:
This is right! The replacement only happens on file write, I think.